src/main/java/com/rapid7/client/dcerpc/msrrp/dto/RegistryValue.java

```diff
@@ -132,7 +135,7 @@ public String getDataAsBinaryStr() {
     }
 
     public String getDataAsHexStr() {
-        return Hex.toHexString(data).toUpperCase();
+        return BaseEncoding.base16().encode(data); //Hex.toHexString(data).toUpperCase();
```

Comment:
remove comment

Should this still uppercase the encoded data or is that defaulted to?

Reply:
it defaults to uppercase
[Base Encoding](https://guava.dev/releases/16.0/api/docs/com/google/common/io/BaseEncoding.html)